front/app/containers/Admin/communityMonitor/components/Settings/components/SurveySettings.tsx

```diff
@@ -0,0 +1,191 @@
+import React, { useState } from 'react';
+
```

Comment:
I felt there were enough differences between the Community Monitor settings and the regular Native Survey settings to justify some repetition rather than making the original settings component more configurable.

So there is definitely overlap between this component + the regular Native Survey settings, but I think it's easier to understand :thinking: